.gitattributes

```diff
@@ -0,0 +1 @@
+internal/mountinfo/fixtures/sys/** linguist-generated=true
```

Comment:
nit: go usually you store this data under a `testdata` directory, not `fixtures`.